movieApp/asset/app.js

```diff
@@ -0,0 +1,50 @@
+//const API_KEY=daf7b11fd7dfe1a364da7a86d79ba727;
+const IMG_URL="https://image.tmdb.org/t/p/w500";
+const url="https://api.themoviedb.org/3/search/movie?api_key=daf7b11fd7dfe1a364da7a86d79ba727";
```

Comment:
you can name these two url as: IMG_URL and MOVIES_URL for instance. But ofc you can leave them as they are, but then just make them consistent - at the moment one name is in capital letters and the other one in small.

Reply:
ok